src/commands/coin/coinAdjust.ts

```diff
@@ -22,15 +30,22 @@ class CoinBalanceCommand extends AdminCommand {
           key: 'amount',
           prompt: 'enter the amount to adjust.',
           type: 'integer'
+        },
+        {
+          key: 'reason',
+          prompt: '',
```

Comment:
prompt?

Reply:
This argument is optional as it has a default value. As such, even when this prompt is not an empty string, the prompt does not appear. Commando gives an error though if the prompt is removed, since it says that a prompt is required, so I just left it as an empty string.

Reply:
makes sense!